CMakeLists.txt

```
@@ -79,14 +78,23 @@ if (CMAKE_SYSTEM_PROCESSOR MATCHES "^(ppc|powerpc)")
   else()
     set(is-ppc 1)
   endif()
-endif ()
+  endif ()
```

This is an accidental whitespace change, isn't it? If so:
```suggestion
endif ()
```